### PR TITLE
feat(Visualization): LinkNeighborloader visualization.

### LINKED ISSUE #55: Visualize the graph that comes out of NeighborLoader

### PR DESCRIPTION
**Pull request by szemyd**
closes #55

### CODE REVIEW

File: data/data_loader.py [hunk 1]
```diff
@@ -53,8 +53,8 @@ def create_dataloaders(
 
     train_loader = LinkNeighborLoader(
         train_split,
-        num_neighbors=[10, 10],
-        batch_size=config.batch_size,
+        num_neighbors=[3],
+        batch_size=15,  # config.batch_size,
```
Comment (itchingpixels) [inline data/data_loader.py:57]:
```suggestion
        batch_size=config.batch_size,
```

File: utils/visualize.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+
+def select_layout(G: Union[nx.DiGraph, nx.Graph], node_types: Optional[list]):
+    one_category_nodes = [
+        node if node_type == "customer" else None
+        for node, node_type in zip(G.nodes(), node_types)
+    ]
+
+    pos = nx.bipartite_layout(G, nodes=one_category_nodes)
+    # pos = nx.kamada_kawai_layout(G)
```
Comment (itchingpixels) [inline utils/visualize.py:22]: can we remove these?:)

File: utils/visualize.py [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    _, edge_types = data.metadata()
+
+    edge_dicts = [data[edge_type] for edge_type in edge_types]
+
+    all_edges = edge_dicts[0].edge_index.detach().clone()
+    edge_label = edge_dicts[0].edge_label.detach().clone()
+    edge_label_index = edge_dicts[0].edge_label_index.detach().clone()
+
+    len_customer_indexes = torch.max(all_edges, dim=1)[0][0].item() + 1
+    edge_label_index[1] = torch.add(edge_label_index[1], len_customer_indexes)
```
Comment (itchingpixels) [inline utils/visualize.py:44]: by the way, you can use `+` here, torch has support for overloaded operators on Tensors, just FYI:)

Comment (itchingpixels) [reply utils/visualize.py:44]: no need to change it of course

File: utils/visualize.py (hunk 2 above)
Comment (szemyd) [reply utils/visualize.py:22]: These are on purposely left in, it's useful to see various versions of the graph. We could of course have an if statement and pass in a selector variable but I think for our current purposes this is faster and more managable.

**Pull request merged**